src/components/organisms/curriculum/OakDownloadsJourneyChildSubjectTierSelector/OakDownloadsJourneyChildSubjectTierSelector.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { StoryObj, Meta } from "@storybook/react";
+
+import { OakDownloadsJourneyChildSubjectTierSelector } from "./OakDownloadsJourneyChildSubjectTierSelector";
+
+const meta: Meta<typeof OakDownloadsJourneyChildSubjectTierSelector> = {
```

Comment:
It's quite good to define the argTypes and controls here to make the story more useable for others.